custom/pnlppgi/reports.py

```diff
@@ -36,6 +36,20 @@ def update_config(config):
     config.update(dict({get_INFilter_element_bindparam('owner_id', idx): val for idx, val in enumerate(users, 0)}))
 
 
+def users_locations():
```

Comment:
can you explain what this is / what it's doing? especially the hard coded group id. might be worth a comment or doc string.